README.md

```diff
@@ -36,6 +36,14 @@ Please consider donating to support me and help me put more time into my plugins
     "autoturret",
     "lmg.m249"
   ],
+  "GUI button": {
+    "Enabled?": true,
+    "Image": "https://i.imgur.com/CyF0QNV.png",
+    "Background color (RGBA format)": "1 1 1 0.15",
+    "AnchorMin": "0.6445 0.025",
+    "AnchorMax": "0.6905 0.107"
```

Comment:
Only using anchors leads to issues with different aspect ratios. The proper way to do this sort of thing where you are drawing relative to existing UI, is to use anchors to start from a specific point, then use offsets to draw the element from there. In this case, the anchor would be set to the bottom center of the screen with both anchor min/max having the same value.

Off the top of my head, the anchor values would be `"0.5 0"`. It's recommended to set to resolution 1280x768 when setting the pixel offsets. They will automatically scale with resolution but won't have stretch issues that anchors do.

Reply:
I don't have much experience doing this at this moment. We can improve this in later release. My variant is pixel-perfect solution for aspect ratio 16:9 (e.g. 1920x1080) which is "standard de facto" at this moment and most common.